tools/README.md

```diff
@@ -11,3 +11,34 @@
 
 Note: On contract deployment, most of the tools (e.g. [ethers](https://docs.ethers.io/v5/api/utils/address/#utils--contract-addresses)) pre-compute the contract address on the client-side, based
 on sender address and nonce. In the Hedera ecosystem, it's not like that, where it's just the next available id.
+
```

Comment:
Update the Note:" to add
"[ethers js](https://docs.ethers.io/v5/api/utils/address/#utils--contract-addresses) and therefore Hardhat usage are impacted by this address calculation difference with the details captured [here](https://github.com/ethers-io/ethers.js/discussions/3141).
An extra step to retrieve the valid Hedera contract address is required to workaround this challenge, example workarounds are provided below"

Reply:
You didn't add the link to the discussion item on the etherejs repo for the "here"